test/suite/test_txn19.py

```diff
@@ -481,7 +482,13 @@ def run_wt_and_check(self, dir, errfile, outfile, expect_fail):
             closeconn=False)
 
         if expect_fail:
-            self.check_file_contains_one_of(errfile, ['WT_TRY_SALVAGE: database corruption detected'])
+            errmsg = 'WT_TRY_SALVAGE: database corruption detected'
+            if self.filename == 'WiredTigerHS.wt':
+                if self.kind == 'removal':
+                    errmsg = 'hs_exists'
```

Comment:
This hits a separate code path now so I had to adjust the error string from `handle-open` to `hs_exists`.